diff --git a/helpers/expoNotifications.js b/helpers/expoNotifications.js
index b7ed3c0..975b716 100644
--- a/helpers/expoNotifications.js
+++ b/helpers/expoNotifications.js
@@ -20,6 +20,9 @@ const sendAllNotifications = (notifications) => {
     const currentUserNotifications = userNotifications[user];
     redis.smembersAsync(`tokens:${user}`)
       .then(async (tokens) => {
+        if (tokens.length === 0) return;
+
+        console.log('tokens', user, tokens);
         const messages = [];
         currentUserNotifications.forEach((currentUserNotification) => {
           tokens.forEach(token => messages.push(
@@ -29,6 +32,7 @@ const sendAllNotifications = (notifications) => {
         const chunks = expo.chunkPushNotifications(messages);
         for (const chunk of chunks) {
           try {
+            console.log(chunk);
             const resp = await expo.sendPushNotificationsAsync(chunk);
             console.log('Expo chunk set', resp);
           } catch (error) {
@@ -89,7 +93,7 @@ const getNotificationMessage = (notification, token) => {
       };
   }
 
-  return { ...template, message };
+  return { ...template, ...message };
 };
 
 module.exports = {
